File: SofiForce.Models/Models/SearchModels/StoreSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class StoreSearchModel : BaseSearchModel
    {
        public int? BranchId {get; set;}
        public int? StoreTypeId { get; set; }
        public int? IsOrder { get; set; }
        public int? IsActive { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/StoreSearchModel.ts

import { BranchModel } from "../EntityModels/branchModel";
import { StoreTypeModel } from "../EntityModels/storeTypeModel";
import { BaseSearchModel } from "./baseSearchModel";

export interface StoreSearchModel extends BaseSearchModel {
    branchId: number | null;
    branchCode: string;
    storeTypeId: number | null;
    isActive: number | null;
    isOrder: number | null;

    storeMode:number;

}